libs/dotcms-models/src/lib/dot-asset-create-options.model.ts

```diff
@@ -0,0 +1,8 @@
+import { DotCMSTempFile } from '@dotcms/dotcms-models';
+
+export interface DotAssetCreateOptions {
+    files: DotCMSTempFile[];
+    updateCallback: (processed: number) => void;
+    url: string;
+    folder: string;
+}
```

Comment:
Make sure that we only add models in `dotcms-models` that are share, if they are very block editor oriented, leave them there.

Reply:
this is the case, it will be used in `dotcms-webcomponents`